R/zchunk_LA114.wind-energy.R

```diff
@@ -1,24 +1,26 @@
 #' module_energy_LA114.wind
 #'
-#' Briefly describe what this chunk does.
+#' This chunk calculates regional supply curves for wind using country-level supply curves.
 #'
 #' @param command API command to execute
 #' @param ... other optional parameters, depending on command
 #' @return Depends on \code{command}: either a vector of required inputs,
 #' a vector of output names, or (if \code{command} is "MAKE") all
 #' the generated outputs: \code{L114.RsrcCurves_EJ_R_wind}. The corresponding file in the
 #' original data system was \code{LA114.wind.R} (energy level1).
-#' @details Describe in detail what this chunk does.
+#' @details Regional supply curves were generated by summing up country-level supply curves and then smoothing them out.
+#' @details Country-level supply curves were generated using data from Zhou. Every country within a region shared the
```

Comment:
Don't need to keep repeating `@details` in successive lines. Just once at the beginning.